CHANGELOG.md

```diff
@@ -7,231 +7,20 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
-- build - refactor build system for easier configuration ([#10718](https://github.com/MetaMask/metamask-extension/pull/10718))
-- ci - cache deps before patch-package ([#10735](https://github.com/MetaMask/metamask-extension/pull/10735))
-- Update @metamask/controllers to v6.2.1 ([#10701](https://github.com/MetaMask/metamask-extension/pull/10701))
-- fix: remove unused `metamask.rpcUrl` from redux state + fix tests to reflect that ([#10714](https://github.com/MetaMask/metamask-extension/pull/10714))
-- Delete setupFetchDebugging.js ([#10636](https://github.com/MetaMask/metamask-extension/pull/10636))
-- deps - remove "remotedev-server" ([#10687](https://github.com/MetaMask/metamask-extension/pull/10687))
-- Adding default properties to NetworkForm ([#10682](https://github.com/MetaMask/metamask-extension/pull/10682))
-- make migration more safe ([#10689](https://github.com/MetaMask/metamask-extension/pull/10689))
-- refactor incoming tx controller ([#10639](https://github.com/MetaMask/metamask-extension/pull/10639))
-- Fixing incorrectly typed token decimal attribute ([#10666](https://github.com/MetaMask/metamask-extension/pull/10666))
-- fix: replace dnode background with JSON-RPC ([#10627](https://github.com/MetaMask/metamask-extension/pull/10627))
-- Improve specificity of `test:unit:lax` npm script ([#10661](https://github.com/MetaMask/metamask-extension/pull/10661))
-- Fixing migration script generation paths ([#10664](https://github.com/MetaMask/metamask-extension/pull/10664))
-- colocate tests in flat structure ([#10655](https://github.com/MetaMask/metamask-extension/pull/10655))
-- Allow TextField to receive min and max attributes ([#10656](https://github.com/MetaMask/metamask-extension/pull/10656))
-- Sync `master` with `develop` ([#10652](https://github.com/MetaMask/metamask-extension/pull/10652))
-- Merge remote-tracking branch 'origin/develop' into sync-master
-- Move swaps constants to the shared constants directory ([#10614](https://github.com/MetaMask/metamask-extension/pull/10614))
-- no more node:console ([#10640](https://github.com/MetaMask/metamask-extension/pull/10640))
-- Setting balance to 0x0 when the original value is undefined ([#10634](https://github.com/MetaMask/metamask-extension/pull/10634))
-- remove transactionCategory in favor of more types ([#10615](https://github.com/MetaMask/metamask-extension/pull/10615))
-- Ensure permission log will only store JSON-able data ([#10524](https://github.com/MetaMask/metamask-extension/pull/10524))
-- Replace logic for eth swap token in fetchQuotesAndSetQuoteState with getSwapsEthToken call ([#10624](https://github.com/MetaMask/metamask-extension/pull/10624))
+### Added
+- [#11021](https://github.com/MetaMask/metamask-extension/pull/11021): Add periodic reminder modal for backing up recovery phrase
+- [#11179](https://github.com/MetaMask/metamask-extension/pull/11179): Add warning when attempting to add a custom network that already exists
```

Comment:
Might be worth clarifying that this change only applies to the custom network form, since "Add Ethereum Chain" already prevented this.

Reply:
Updated this log entry 👍